app/src/main/java/com/example/atheneum/activities/MainActivity.java

```diff
@@ -183,7 +168,27 @@ public void onCancelled(@NonNull DatabaseError databaseError) {
 
                 }
             });
+        } else if (id == R.id.nav_notifications) {
+            final Intent notifications_intent = new Intent(this, NotificationsActivity.class);
 
+            FirebaseUser firebaseUser = FirebaseAuthUtils.getCurrentUser();
+            FirebaseDatabase db = FirebaseDatabase.getInstance();
+            DatabaseReference dbRef = db.getReference("users").child(firebaseUser.getUid());
+

```

Comment:
Kind of a minor thing but I have a viewmodel in the MainActivity that is tied to the firebaseUser UID so you could use that instead here by making the viewmodel a private variable of the activity and then using it here. If you need to simulate a single value event you can remove the LiveData observer after the data is retrieved.

Reply:
Oh, I forgot to remove this. Removed in future commit